.travis.yml

```diff
@@ -34,9 +34,10 @@ install:
   - $PYTHON_EXE -m easy_install --version
   - $PYTHON_EXE -m pip --version
   - $PYTHON_EXE -m tox --version
-  - $PYTHON_EXE -m pip install -U setuptools
```

Comment:
But why? Now I actually noticed what is happening, this doubles the tests we are running just for testing with older setuptools versions?! Sorry this makes no sense whatsoever.